source/python/neuropod/backends/python/executor.py

```diff
@@ -73,6 +74,15 @@ def __init__(self, neuropod_path, load_custom_ops=True):
         entrypoint_package_path = model_config["entrypoint_package"]
         entrypoint_fn_name = model_config["entrypoint"]
 
+        if is_native:
```

Comment:
in case of is_native = False, do you expect that requirements.lock should be installed manually?

Reply:
Added more details in the comment:

```
# For other code paths (e.g. when calling `load_neuropod` from python with `_always_use_native=False`),
# the user is responsible for ensuring that all dependencies are installed in the environment
```